packages/fluentui/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@fluentui/noop",
+  "private": true,
+  "version": "0.0.0"
```

Comment:
https://github.com/lerna/lerna/blob/6c4ee5294002c0e464f557b149516017eccf2a07/core/project/index.js#L101

Is required as we should have `package.json` under the same directory